rusoto/core/src/request.rs

```diff
@@ -35,14 +35,19 @@ lazy_static! {
             env!("CARGO_PKG_VERSION"), RUST_VERSION, env::consts::OS).as_bytes().to_vec()];
 }
 
-
+/// Stores the response from a HTTP server.
 pub struct HttpResponse {
+    /// Status code of HTTP Request
     pub status: StatusCode,
+    /// Contents of Response
     pub body: Box<Read>,
+    /// Headers stored as <key:content>
     pub headers: HashMap<String, String>,
 }
 
 #[derive(Debug, PartialEq)]
+/// An error produced when invalid request types are sent.
+/// Valid examples are `GET`, `POST`, `PUT` etc.
```

Comment:
This valid examples comment confuses me here. We're talking about errors, so why are valid examples listed?